codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ViewServiceSourceGenerator.scala

```diff
@@ -142,13 +142,13 @@ object ViewServiceSourceGenerator {
         |    new ${view.providerName}(viewFactory, viewId)
         |
         |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-        |    ${view.fqn.parent.name}Proto.javaDescriptor.findServiceByName("${view.fqn.protoName}")
+        |    ${typeName(view.descriptorObject)}.javaDescriptor.findServiceByName("${view.fqn.protoName}")
```

Comment:
no change in ViewServiceSourceGeneratorSuite?

Reply:
no, I suspect thats due to `TestData` not being entirely realistic still, not sure if it's worth digging into the details

Reply:
I see, it's defined as `my_service.proto`, which accidentally is the same as MyService